src/main/resources/db.changelog/db.changelog-master.yaml

```
@@ -1,3 +1,15 @@
 databaseChangeLog:
   - include:
-      file: db/changelog/changes/01-create-book-table.yaml
\ No newline at end of file
+      file: db/changelog/changes/01-create-book-table.yaml
+  - include:
+        file: db/changelog/changes/02-create-user-table.yaml
+  - include:
+        file: db/changelog/changes/03-insert-user.yaml
+    - include:
+        file: db/changelog/changes/04-create-role-table.yaml
+    - include:
+        file: db/changelog/changes/05-insert-role.yaml
+    - include:
+        file: db/changelog/changes/06-create-users-roles-table.yaml
+    - include:
+        file: db/changelog/changes/07-insert-roles-data.yaml
```

Don't forget to change file names
```suggestion
        file: db/changelog/changes/02-create-user-table.yaml
  - include:
        file: db/changelog/changes/03-insert-users.yaml
    - include:
        file: db/changelog/changes/04-create-role-table.yaml
    - include:
        file: db/changelog/changes/05-insert-roles.yaml
    - include:
        file: db/changelog/changes/06-create-users-roles-table.yaml
    - include:
        file: db/changelog/changes/07-insert-users-roles.yaml
```